src/pages/user/Settings.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { UserSettings } from './Settings';
+import { withMemoryRouter } from 'utils/testing';
+
+describe('<UserSettings />', () => {
+  const actions = {
+    toggleDarkMode: jest.fn()
+  };
+  const props = {
+    theme: 'default',
+    actions
+  };
+
+  const RoutedUserSettings = withMemoryRouter(UserSettings);
```

Comment:
Does this test give warnings/errors if you remove the `withMemoryRouter`?

Reply:
Ah, nope. Removed!